about.html

```diff
@@ -12,7 +12,7 @@
             <section id="about" class="visible" data-title="About">
                 <a href="./" class="logo">
                     <img srcset="resources/logo@2x.png 2x" src="resources/logo.png" alt="Speedometer" />
-                    <div class="version">3.0</div>
+                    <div class="version next">next</div>
                 </a>
                 <div class="section-grid">
                     <h1 class="section-header">About Speedometer 3</h1>
```

Comment:
This specific one is still there :-)  But maybe we can remove the version completely for this line, what do you think?